app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sponsors/SponsorsViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package io.github.droidkaigi.confsched2018.presentation.sponsors
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.ViewModel
+import io.github.droidkaigi.confsched2018.data.repository.SponsorPlanRepository
+import io.github.droidkaigi.confsched2018.model.SponsorPlan
+import io.github.droidkaigi.confsched2018.presentation.Result
+import io.github.droidkaigi.confsched2018.presentation.common.mapper.toResult
+import io.github.droidkaigi.confsched2018.util.ext.toLiveData
+import io.github.droidkaigi.confsched2018.util.rx.SchedulerProvider
+import io.reactivex.disposables.CompositeDisposable
+import javax.inject.Inject
+
+class SponsorsViewModel @Inject constructor(
+        private val sponsorPlanRepository: SponsorPlanRepository,
+        private val schedulerProvider: SchedulerProvider
+) : ViewModel() {
+    private val compositeDisposable: CompositeDisposable = CompositeDisposable()
```

Comment:
This should be removed too?

Reply:
oh, that's true! thanks!

Reply:
This was re-born because of https://github.com/DroidKaigi/conference-app-2018/pull/176/commits/e4ef1e4adba92dec4c78ca60b75d6dd42c075806 🙏